lib/errors.ts

```
@@ -0,0 +1,11 @@
+class ChatError extends Error {
+  constructor(message: string, public code: string,{
+    
+  }) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)